geomesa-fs/geomesa-fs-dist/pom.xml

```diff
@@ -42,76 +40,34 @@
             <artifactId>geomesa-fs-storage-orc_${scala.binary.version}</artifactId>
         </dependency>
 
-        <!-- distributed runtime -->
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-fs-spark-runtime_${scala.binary.version}</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-fs-spark_${scala.binary.version}</artifactId>
-        </dependency>
-
-        <!-- geoserver plugins -->
+        <!-- geoserver plugin - needed as a dependency for reactor ordering, but don't bring in transitive deps -->
         <dependency>
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-fs-gs-plugin_${scala.binary.version}</artifactId>
             <type>pom</type>
-            <!-- Don't bring any any transitives of dist tar.gz files -->
+            <scope>provided</scope>
             <exclusions>
                 <exclusion>
                     <groupId>*</groupId>
                     <artifactId>*</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-process-wps_${scala.binary.version}</artifactId>
-        </dependency>
 
-        <!-- additional deps to manage in for deploying to lib dir-->
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-fs-datastore_${scala.binary.version}</artifactId>
-        </dependency>
+        <!--spark  runtime -->
```

Comment:
Does depending on the Spark runtime get this pom to the right spot in the reactor?  Either way, it may be worth sticking the pattern you've established.

Reply:
i think all the modules depend on the associated spark runtime, as they get bundled into the dist/spark dir. do you mean we shouldn't attach them?

Reply:
No the spark runtimes should be in the dist bundles.  I was trying to suggest that maybe depending on the gs-plugin and then excluding it was an unnecessary trick?

Reply:
i think i had to exclude the transitive deps from the gs-plugin b/c it pulls in some unnecessary jars, but didn't have to do the same with spark i guess... there was definitely a reason i did it haha

Reply:
:shipit: